src/main/java/com/volmit/iris/util/plugin/VolmitSender.java

```diff
@@ -423,55 +425,99 @@ public void sendDecreeHelp(VirtualDecreeCommand v) {
 
     public void sendDecreeHelpNode(VirtualDecreeCommand i){
         if (isPlayer()) {
```

Comment:
Explain what you changed because it looks like it was just formatted, keep it the old way unless there's a good reason for it.

Reply:
Reformat only.
Behaves exactly the same except for a minor performance drop in exchange for actual readability and comprehensibility.